python_modules/dagster-graphql/dagster_graphql/implementation/events.py

```diff
@@ -168,6 +169,7 @@ def iterate_metadata_entries(metadata: Mapping[str, MetadataValue]) -> Iterator[
                         label=reference.label,
                     )
                     for reference in value.code_references
+                    if isinstance(reference, LocalFileCodeReference)
```

Comment:
Is the idea that this condition is temporary until we add support at the graphql layer for remote code references?

Reply:
yes

Reply:
👍